README.md

```diff
@@ -1 +1,3 @@
-# RealTimeGarbageCollector
\ No newline at end of file
+# RealTimeGarbageCollector
+
+IFT611 project.
```

Comment:
Please add a complete summary of your project. This would help future viewers wanting to help.